src/main/java/frc/robot/commands/OuttakeMotorTest.java

```diff
@@ -3,21 +3,23 @@
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot.commands;
+
 import frc.robot.subsystems.IntakeSubsystem;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 public class OuttakeMotorTest extends CommandBase {
-  IntakeSubsystem Intake = new IntakeSubsystem();
+  IntakeSubsystem Intake;
   /** Creates a new IntakeMotorTest. */
   public OuttakeMotorTest(IntakeSubsystem subsystem) {
```

Comment:
Whats the difference between the intake and the outtake, this file should at least have differently named stuff probably?